apps/total-typescript/src/ability/ability.ts

```diff
@@ -34,7 +39,7 @@ type ViewerAbilityInput = {
   user?: User
   subscriber?: any
   lesson?: Exercise
-  module: SanityDocument
+  module?: SanityDocument
```

Comment:
this was tripping it up since we are on a different page, probably deserves a rethink in terms of "rules factories"